slsim/Sources/random_supernovae.py

```diff
@@ -54,10 +79,10 @@ def __init__(
         :type absolute_mag_band: str or `~sncosmo.Bandpass`
         :param mag_zpsys: Optional, AB or Vega (AB default)
         :type mag_zpsys: str
+        :param modeldir: Path to the directory containing supernova files
+        :type modeldir: str
         :param cosmo: Cosmology for absolute magnitude
         :type cosmo: `~astropy.cosmology`
-        :param random_seed: Random seed for randomization
```

Comment:
is there a reason to remove the random seed argument. This could help for reproducing a specific realization of the random draw

Reply:
The random seed argument was already removed. I just removed the comment for it.

Reply:
Maybe good to put back this random seed for reproducibility.